gauntlet/packages/gauntlet-serum-multisig/src/commands/multisig.ts

```diff
@@ -95,44 +102,46 @@ export const wrapCommand = (command) => {
       return result
     }
 
-    wrapAction = (action: ProposalAction) => async (proposal: Keypair | PublicKey, context: ProposalContext) => {
-      try {
-        const tx = await action(proposal, context)
-        return {
-          responses: [
-            {
-              tx: this.wrapResponse(tx, proposal.toString()),
-              contract: proposal.toString(),
-            },
-          ],
-        } as Result<TransactionResponse>
-      } catch (e) {
-        // known errors, defined in multisig contract. see serum_multisig.json
-        if (e.code >= 300 && e.code < 400) {
-          logger.error(e.msg)
```

Comment:
Why removing this logging for known erros?

Reply:
This wasn't doing anything. The program already logs this errors automatically